atlas_densities/densities/mtype_densities_from_map/create.py

```diff
@@ -73,62 +70,38 @@ def create_from_probability_map(
             - the sum of each row of the probability map is different from 1.0
             - the labels of the rows and columns of `probablity_map` are not all lowercase
             or contain some white spaces.
-            - the layer names appearing in the row labels of `probability_map` are not those
-            described in `metadata`
     """
     # pylint: disable=too-many-locals
-    region_ids = []
-    for region in metadata["regions"]:
```

Comment:
the metadata also had a list of regions for which to compute the densities, should it now just compute everything all the time?

Reply:
I think it should just use what is in the config file, and assume that it's the ground truth.